app/javascript/apps/ArtsyPullRequests/ArtsyPullRequests.spec.tsx

```diff
@@ -39,12 +36,12 @@ describe("rendering", () => {
       pullRequests: [pullRequest],
     }
 
-    const wrapper = mount(<ArtsyPullRequests {...props} />)
+    render(<ArtsyPullRequests {...props} />)
 
-    expect(wrapper.find(PullItem)).toHaveLength(1)
+    expect(screen.queryByRole("link")).toHaveTextContent(pullRequest.title)
```

Comment:
Again, `queryBy` will throw if there are multiple, so this test is still validating that there's only 1. Also there's some extra assertions like checking for the presence of some text which are really handy.